src/app/api/books/recommend/route.ts

```diff
@@ -0,0 +1,40 @@
+import { PrismaClient } from "@prisma/client";
+import { NextResponse } from "next/server";
+
+const prisma = new PrismaClient();
```

Comment:
Use prisma from "@/lib/prisma"
```suggestion
import { prisma } from "@/lib/prisma";
```